models/wrf_hydro/matlab/plot_connections.m

```diff
@@ -1,8 +1,8 @@
-function plot_connections(x, ROUTE_FILE, fig_pos, show_gauges_legend, colorbar_title)
+function plot_connections(x, ROUTE_FILE, fig_pos, colorbar_title, tiny_flow)
 
 %% DART software - Copyright UCAR. This open source software is provided
 % by UCAR, "as is", without charge, subject to all terms of use at
-% http://www.image.ucar.edu/DAReS/DART/DART_download
+% https://dart.ucar.edu/software/
```

Comment:
```suggestion
% http://www.image.ucar.edu/DAReS/DART/DART_download
```

The terms of use are at http://www.image.ucar.edu/DAReS/DART/DART_download

Reply:
Done